geode-core/src/test/java/org/apache/geode/internal/net/NioSslEngineTest.java

```diff
@@ -411,7 +417,7 @@ public void closeWhenSocketWriteError() throws Exception {
         // give the NioSslEngine something to write on its socket channel, simulating a TLS close
         // message
         outputSharing.getBuffer().put("Goodbye cruel world".getBytes());
-        return new SSLEngineResult(CLOSED, FINISHED, 0, 0);
+        return new SSLEngineResult(OK, NEED_UNWRAP, 0, 0);
```

Comment:
The result value here (`OK, NEED_UNWRAP`) was empirically derived. I don't believe changing this value changes the intention of the test (`closeWhenSocketWriteError()`), which was/is to verify that when `close()` encounters a `ClosedChannelException` when calling `SocketChannel.write()`, that no exception is thrown (from `close()`).